Crashlytics/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # Unreleased
 - [changed] Removed obsolete API calls from upload-symbols
+- [changed] Removed unneccessary onboarding calls from the SDK.
```

Comment:
super duper nit: maybe just match the wording of the upload-symbols change above `...obsolete API...`